app/javascript/store/modules/froggo_team/actions.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+import {
+    CREATE_NEW_FROGGO_TEAM,
+} from '../../action-types';
+
+export default {
+  [CREATE_NEW_FROGGO_TEAM](_,{ name, organization_id, users }) {
+    axios.post(
+      `/api/organizations/${organization_id}/froggo_teams`,
```

Comment:
Identifier 'organization_id' is not in camel case                                                              camelcase